lib/features/steps/runner_steps.rb

```diff
@@ -8,6 +8,21 @@
   Runner.environment[key] = value
 end
 
+# Verifies that an environment variable is set to a given value.
+#
+# @step_input key [String] The environment variable
+# @step_input value [String] The intended value of the environment variable
+Then('the environment variable {string} equals {string}') do |key, expected_value|
+  assert_equal(expected_value, Runner.environment[key])
+end
+
+# Verifies that an environment variable is not set.
+#
+# @step_input key [String] The environment variable
+Then('the environment variable {string} is null') do |key|
+  assert_nil(Runner.environment[key])
+end
```

Comment:
```suggestion
Then('the environment variable {string} is null') do |key|
  assert_nil(ENV[key])
end
```

or if this should check that it's not set then maybe this would be better:

```suggestion
Then('the environment variable {string} is not set') do |key|
  assert_false(ENV.key?(key))
end
```

Reply:
Discussed IRL.  New steps (which are only intended for the verification of MazeRunner functionality) have been renamed and pushed into the fixture directory.